docs/platform/howto/password-policy.rst

```diff
@@ -0,0 +1,48 @@
+Password Policy
+===============
+
+Aiven is committed to enhancing security of our customers and preventing hackers from accessing your account. Creating a strong password makes it harder for attackers to gain unauthorized access to your account.
+
+**Password Requirements**
+
+To ensure your password is safer, Aiven enforces a number of password criteria that make sure that the password is strong:
+
+-  Minimum password length is 8 characters
+
+-  Cannot contain single repeating characters such as aaaaaaaa 
+
+-  Cannot contain your name or email address
+
+-  Cannot contain common words, phrases or strings such as password, security, John . Using common words or service names can be easily guessed.
+
+-  Cannot contain words that are very similar to common words such as password1.
+
+For **new user** sign ups, these requirements are automatically enforced.
+
+For **existing customers**, while it is not required to change password immediately, it is strongly recommended to update your existing password. When changing your password, the above requirements will be enforced.
+
+When creating **integration** passwords on Aiven's end, the above requirements are also applied. For remote services such as sending logs to external OpenSearch®, the requirements are not applied, but are recommended.
+
+Additionally, when creating and/or resetting the password, please consider the following tips:
+
+**Password Tips**
+
+-  Use password manager to create a randomly generated strong password
+
+-  Use passphrases instead of passwords which are even harder to guess
+
+-  Do not use same password for multiple services
+
+-  Select a password that is not usual and memorable only to you
+
+Selecting a strong password is a first step in securing your account. In addition to creating a secure password, please refer :doc:`./user-2fa`.
+
+If you forget your password, you may reset your password by selecting ``Forgot Password``  on the `Aiven web console login <https://console.aiven.io/>`_ page.
+
+If you have any questions, please feel free to contact the support team via ``support@aiven.io`` to assist you.
```

Comment:
Not sure if we should always have this at the end of the docs, or only for important things

```suggestion
If you have any questions, please feel free to contact the support team via support@Aiven.io to assist you.
```